### PR TITLE
Introduce a query for obtaining environments used to solve a package

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/user-api/issues/1539

## This introduces a breaking change

- [x] No

### CODE REVIEW

**Comment by fridex** [comment]
/assign @pacospace

PTAL. I'm also unsure about the naming.

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -888,6 +888,54 @@ def get_solved_python_packages_all_versions(
...
+        start_offset: int = 0,
+        count: Optional[int] = DEFAULT_COUNT,
+        distinct: bool = False,
+    ) -> List[Dict[str, str]]:
+        """Retrieve all the environments that were used to solve the given package.
+
+        Examples:
+        >>> from thoth.storages import GraphDatabase
+        >>> graph = GraphDatabase()
+        >>> graph.get_solved_python_package_environments_all( \
```
Comment (pacospace) [inline thoth/storages/graph/postgres.py:906]: get_solved_python_package_version_environments_all

**Review by pacospace — COMMENTED** [review]
lgtm! one small thing

File: thoth/storages/graph/postgres.py (hunk 1 above)
Comment (fridex) [reply thoth/storages/graph/postgres.py]: Thanks, adjusted 👍🏻

**Review by pacospace — APPROVED** [review]
lgtm! 💯

**Pull request merged**